tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml

```diff
@@ -581,6 +581,19 @@ platform_tests/broadcom:
     conditions:
       - "'marvell' in asic_type"
 
+platform_tests/broadcom/test_ser.py:
```

Comment:
test_ser should be the only case in test_ser.py, may we combine these two rules?
like conditions_logical_operator: 'OR'

Reply:
you're right! I combined 2 skips under a single file section